### ISSUE TITLE
Indicate that Python 2.4 is not supported on Windows in a document somewhere

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4705]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4705|
|Type|enhancement|
|Created|2010-10-19 15:14:25Z|
|Branch|https://github.com/twisted/twisted/tree/no-win-2.4-4705|

We said we were dropping Python 2.4/Windows support when we released 10.1, meaning we're not claiming to support it in 10.2.  The README should be updated to reflect this.

<details><summary>Searchable metadata</summary>

```
trac-id__4705 4705
type__enhancement enhancement
reporter__exarkun exarkun
priority__high high
milestone__Twisted_10_2 Twisted 10.2
branch__branches_no_win_2_4_4705 branches/no-win-2.4-4705
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1287501265000000 1287501265000000
changetime__1290485390000000 1290485390000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_1"></a>|@glyph commented|
|-|-|

(In [30351]) Branching to 'no-win-2.4-4705'

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_2"></a>|@glyph set owner to @exarkun|
|-|-|

Changes in INSTALL & README.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @glyph|
|-|-|

Looks good, thanks, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set status to `closed`|
|-|-|

(In [30354]) We don't support Python 2.4 on Windows.

Author: glyph

Reviewer: exarkun

Fixes: [#4705](https://github.com/twisted/twisted/issues/4705)

Update the README and INSTALL files to note that we don't support Python 2.4 on
Windows (and adjust some awkward phraseology where we describe Twisted in the
README.)

Update the README and INSTALL files to note that w

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
